pkgs/development/python-modules/abjad/default.nix

```
@@ -0,0 +1,42 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, ply
+, roman
+, uqbar
+}:
+
+buildPythonPackage rec {
+  pname = "abjad";
+  version = "3.19";
```

```suggestion
  version = "3.19";
  format = "setuptools";
  
  disabled = pythonOlder "3.10";
```
Please add `format` and `disabled`.
https://github.com/Abjad/abjad/blob/cead1ac794be198b22d6653a3c6d6baac2553857/setup.py#L99